src/routes/github/create-branch/github-create-branch-get.ts

```diff
@@ -17,10 +17,16 @@ export const GithubCreateBranchGet = async (req: Request, res: Response, next: N
 		return next(new Error(Errors.MISSING_GITHUB_TOKEN));
 	}
 
+	const { issue_key: key, issue_summary: summary } = req.query;
+	if (!key ||!summary) {
```

Comment:
/nit
From UX's point of view, if somehow key || summary is missing, it wouldn't be too bad to still show a page with blank prefill, instead of throwing error to user?

Reply:
We can ignore summary since we are not using it, but I think we need the key.  Will remove the `summary` part.